ledfx/devices/packets.py

```diff
@@ -183,3 +183,16 @@ def build_openrgb_packet(
     out[:, 0:3] = data.astype(np.dtype("B"))
     packet.extend(out.flatten().tobytes())
     return packet
+
+
+def build_launchpad_packet(data: np.ndarray):
+    """
+    Launchpad packet encoding
+
+    """
+    frame_size = len(data)
+
+    # ToDo: this is just a dummy function
```

Comment:
Use capital TODO or FIXME both pycharm supports naitively and I think there is a TODO tree plugin for vscode support that syntax with extra tooling

Reply:
done